src/plugins/gimbal/gimbal_device.cpp

```diff
@@ -0,0 +1,54 @@
+#include "plugins/gimbal/gimbal_device.h"
+#include "gimbal_device_impl.h"
+
+namespace mavsdk {
+
+GimbalDevice::GimbalDevice(System& system) : PluginBase(), _impl{new GimbalDeviceImpl(system)} {}
+
+GimbalDevice::~GimbalDevice() {}
+
+GimbalDevice::Result GimbalDevice::set_attitude(int operation_flags,
+        float quat_w, float quat_x, float quat_y, float quat_z,
```

Comment:
Check how other plugins use `struct`s to pass data like a quaternion in. That many parameters to a function are confusing and dangerous, in my opinion.

Reply:
Okay, will do!